config/default.json

```diff
@@ -32,8 +32,8 @@
 	},
 	"COOKIE": {
 		"SAME_SITE": "none",
-		"HTTP_ONLY": true,
```

Comment:
It exist differents to the client in main and in this pr.
https://github.com/hpi-schul-cloud/schulcloud-client/blob/main/config/default.json#L9

Reply:
You said that the default.schema.json should contain the production values. COOKIE is an object and this apparently has to be defined in the default.json too in order to work correctly. The client uses the production.json for the these values. https://github.com/hpi-schul-cloud/schulcloud-client/blob/main/config/production.json

Reply:
es macht wahrscheinlich sinn das im client auch glattzuziehen, dass dort nicht teile der COOKIE variable in unterschiedlichen files gesetzt werden (dieses production file ist unvollständig)

Reply:
You are true, production.json is deprecated in my mind and i forget zu check it.